docs/_includes/customizer-variables.html

```diff
@@ -2689,7 +2689,7 @@ <h2 id="badges">Badges</h2>
   </div>
   <div class="col-xs-4">
     <label for="input-@badge-border-radius">@badge-border-radius</label>
-    <input id="input-@badge-border-radius" type="text" value="10px" data-var="@badge-border-radius" class="form-control"/>
+    <input id="input-@badge-border-radius" type="text" value="3px" data-var="@badge-border-radius" class="form-control"/>
```

Comment:
Why do we change badge border radius?

Reply:
I ran some build-y stuff? Guessing it was outdated